src/helpers/messages.js

```diff
@@ -38,7 +38,7 @@ function getExitMessage(scope, runner, message = '', sync, submitButtonLabel) {
     if (messageEnabled) {
         itemsCountMessage = getUnansweredItemsWarning(scope, runner, sync);
 
-        if (itemsCountMessage) {
+        if (itemsCountMessage.length > 1) {
```

Comment:
This may not work as expected.

I'd suggest keeping the previous code, but adding a `trim()`